scipy/spatial/_spherical_voronoi.py

```diff
@@ -214,6 +223,8 @@ def __init__(self, points, radius=None, center=None):
             self.radius = radius
         else:
             self.radius = 1
+        if pdist(self.points).max() > (2. * self.radius):
+            raise ValueError("Radius inconsistent with generators.")
```

Comment:
Can this test be easily triggered by fp error?
Would it be better to compute the distance of the points?
Is there an assumption that the sphere is centered at zero?